pages/docs/tutorials/run-rao.md

```diff
@@ -0,0 +1,436 @@
+be provided alongside the impacted network element(s).
+
+> For our example, we only create one contingency corresponding to an outage on the line _FFR1AA1 FFR2AA1_.
+
+```
+crac.newContingency()
+        .withId("contingency")
+        .withNetworkElement("FFR1AA1  FFR2AA1")
+        .add();
+```
+
+### Add instants
+
+Once the contingencies are added, we can now create the different [instants](/docs/input-data/crac/json#instants-states)
+of the optimisation process. An instant is added thanks to the `newInstant` method. Both an identifier and
+an `InstantKind` (`PREVENTIVE`, `OUTAGE`, `AUTO` or `CURATIVE`) must be provided. The instants must also be declared in
+**chronological order**.
+
+> For our example, we will create 4 instants (one of each kind).
+
+```
+crac.newInstant("preventive", InstantKind.PREVENTIVE)
+        .newInstant("outage", InstantKind.OUTAGE)
+        .newInstant("auto", InstantKind.AUTO)
+        .newInstant("curative", InstantKind.CURATIVE);
+```
+
+Now that contingencies and instants are all set, we can start adding CNECs and remedial actions to the CRAC.
+
+### Create FlowCNECs
+
+The next step is to create the [CNECs](/docs/input-data/crac/json#cnecs). For our example and given the simple network
+we are using, we will only consider [FlowCNECs](/docs/input-data/crac/json#flow-cnecs) that correspond to lines in the
+network that will have to be optimised flow-wise after contingencies (and in the base case).
+
+The `newFlowCnec` method needs to be called to create a FlowCNEC in the CRAC. Different information must be provided
+including the CNEC's identifier, network element, instant and thresholds. For non-preventive CNECs, an associated
+contingency must also be provided.
+
+> In our example, we define 3 FlowCNECs:
+> - one at the preventive instant on line _FFR1AA1 FFR2AA1 1_
+> - one at the auto instant on line _BBE2AA1 BBE3AA1 1_
+> - one at the curative instant on line _BBE2AA1 BBE3AA1 1_
+
+```
+crac.newFlowCnec()
+    .withId("FFR1AA1  FFR2AA1  1 - preventive")
+    .withInstant("preventive")
+    .withNetworkElement("FFR1AA1  FFR2AA1  1")
+    .newThreshold()
+        .withMin(-5000d)
+        .withMax(+5000d)
+        .withUnit(Unit.MEGAWATT)
+        .withSide(Side.RIGHT)
+        .add()
+    .newThreshold()
+        .withMin(-5000d)
+        .withMax(+5000d)
+        .withUnit(Unit.MEGAWATT)
+        .withSide(Side.LEFT)
+        .add()
+    .add();
+
+crac.newFlowCnec()
+    .withId("BBE2AA1  BBE3AA1  1 - auto")
+    .withInstant("auto")
+    .withContingency("contingency")
+    .withNetworkElement("BBE2AA1  BBE3AA1  1")
+    .newThreshold()
+        .withMin(-5000d)
+        .withMax(+5000d)
+        .withUnit(Unit.MEGAWATT)
+        .withSide(Side.RIGHT)
+        .add()
+    .newThreshold()
+        .withMin(-7500d)
+        .withMax(+7500d)
+        .withUnit(Unit.MEGAWATT)
+        .withSide(Side.LEFT)
+        .add()
+    .add();
```

Comment:
add an outage CNEC.
increase the threshold on both sides so it resembles a real-life TATL (while preventive and curative would be PATLs)

if running the RAO in DC mode (would be easier to understand for the tutorial), set threshold on one side only